documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -91,7 +94,7 @@ kubectl logs -f _CLUSTER-OPERATOR-POD-NAME_ | grep -E "Kafka version upgrade fro
 +
 [source,shell,subs=+quotes]
```

Comment:
The log message mentioned on line 92 does not exist anymore. It should be removed.

Reply:
Not done. I'd rather change this in a subsequent PR for multi-version upgrades.